src/main/java/de/kaizencode/tchaikovsky/speaker/remote/RemoteSpeaker.java

```diff
@@ -131,8 +135,24 @@ public void setSessionTimeout(int timeoutInSec) {
     public Volume volume() {
         return volume;
     }
+    
+    @Override
+    public void createZone(List<Speaker> speakerItems) throws SpeakerException {
```

Comment:
What do you think about putting createZone and releaseZone into the RemoteZoneManager class and just returning the ZoneManager interface instead? (Similar as done with the Volume interface). This way, users have access to all methods of the ZoneManager.